src/chttpd/test/eunit/chttpd_security_tests.erl

```diff
@@ -236,14 +237,12 @@ should_disallow_anonymous_db_view_cleanup([Url,_UsersUrl]) ->
     ?_assertEqual(<<"unauthorized">>, ErrType).
 
 should_allow_admin_purge([Url,_UsersUrl]) ->
```

Comment:
Should we do the `_disabled` thing? Alternatively, does eunit have a notion of skipped tests?

Reply:
Gonna say samesies for all tests switched to 501 assertions since we'll need to undo them once those features are actually implemented.

Reply:
Was on the fence about this one since it's not really testing _purge functionality but testing security so I thought we'd keep letting the tests run in the meantime before we implement it. But I don't think we're gaining a lot more coverage here anyway so will disable them.

Reply:
I think the easiest thing would be to comment out the tests in the eunit test definition block at the top. There is no skip / or disable switch in eunit that I could find.